### PR TITLE
Added tests to a module that was lacking tests

### PR DESCRIPTION
**Pull request by benrs**
Added new tests to the rating module which previously was testless.

### CODE REVIEW

File: modules/rating/src/test/RatingRangeTest.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import lila.rating.RatingRange
+import org.specs2.mutable._
+
+class RatingRangeTest extends Specification {
+  "RatingRange" should {
+    "set a proper default" in {
+      val rr = new RatingRange(1000, 2000);
```
Comment (jacobrs) [inline modules/rating/src/test/RatingRangeTest.scala]: Case classes don't need the `new` keyword.

File: modules/rating/src/test/RatingRangeTest.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+      rr.toString must_== s"1000-2000"
+    }
+
+    "properly contain a value within its range" in {
+      val rr = new RatingRange(1499, 1501);
+      rr.contains(1500) must_== true
```
Comment (jacobrs) [inline modules/rating/src/test/RatingRangeTest.scala]: cleaner statement is `must beTrue`

File: modules/rating/src/test/RatingRangeTest.scala [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import lila.rating.RatingRange
+import org.specs2.mutable._
+
+class RatingRangeTest extends Specification {
+  "RatingRange" should {
+    "set a proper default" in {
+      val rr = new RatingRange(1000, 2000);
+
+      rr.toString must_== s"1000-2000"
```
Comment (jacobrs) [inline modules/rating/src/test/RatingRangeTest.scala]: you can use must beEqualTo("1000-2000") but this works too, just wondering why you put the s in front of the string, isn't that mostly for templating?

Comment (benrs) [reply modules/rating/src/test/RatingRangeTest.scala]: Was being funky when I initially checked against normal strings, was using a different check at the time though. Will change it.

**Review by jacobrs — APPROVED** [review]

**Review by samantha-kerr — APPROVED** [review]

**Pull request merged**